01-introduction/01-02_generative-design/01-02-01_what-is-generative-design.md

```diff
@@ -15,6 +13,7 @@ We see generative design as:
 Generative design is a specific application of the computational design approach, with the following distinctions:
 
 * The designer defines goals to achieve a design \(rather than the exact steps\).
-* The computer generates many designs \(not just one\).
-* The computer finds a set of optimal solutions that satisfy the designer’s multiple competing goals.
+* The computer helps designer to explore the design space and generate multiple design options \(not just one\).
+* The computer enables designer to find a set of optimal solutions that satisfies multiple comepeting goals.
+* The designer evaluates and compares multiple design scenarios to find a set of design options that fits design goals.
```

Comment:
The designer evaluates and compares multiple design scenarios to find a set of design options that fits THE design goals.    .....but also I agree with Radu's comments. Could we just leave it out?